cmd/dfget/app/root.go

```diff
@@ -135,6 +155,24 @@ func initProperties() {
 	if cfg.ClientQueueSize == 0 {
 		cfg.ClientQueueSize = properties.ClientQueueSize
 	}
+
+	currentUser, err := user.Current()
+	if err != nil {
+		printer.Println(fmt.Sprintf("get user error: %s", err))
+		os.Exit(config.CodeGetUserError)
+	}
+	cfg.User = currentUser.Username
```

Comment:
In fact, `cfg.User` is not a necassary field. I will open a new PR to fix that. Make a record here.